sdk/iot/azure-iot-modelsrepository/setup.py

```diff
@@ -0,0 +1,82 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import re
+from setuptools import setup, find_packages
+
+# azure v0.x is not compatible with this package
+# azure v0.x used to have a __version__ attribute (newer versions don't)
+try:
+    import azure
+
+    try:
+        ver = azure.__version__
+        raise Exception(
+            "This package is incompatible with azure=={}. ".format(ver)
+            + 'Uninstall it with "pip uninstall azure".'
+        )
+    except AttributeError:
+        pass
+except ImportError:
+    pass
+
+
+# Fetch description
+with open("README.md", "r") as fh:
+    _long_description = fh.read()
+
+
+# Fetch version
+with open("azure/iot/modelsrepository/_constants.py", "r") as fh:
+    VERSION = re.search(r"^VERSION\s=\s*[\"']([^\"']*)", fh.read(), re.MULTILINE).group(1)
+if not VERSION:
+    raise RuntimeError("Cannot find version information")
+
+
+setup(
+    name="azure-iot-modelsrepository",
+    version=VERSION,
+    description="Microsoft Azure IoT Models Repository Library",
+    license="MIT License",
+    author="Microsoft Corporation",
+    author_email="azpysdkhelp@microsoft.com",
+    url="https://github.com/Azure/azure-sdk-for-python",
+    long_description=_long_description,
+    long_description_content_type="text/markdown",
+    classifiers=[
+        "Development Status :: 3 - Alpha",
```

Comment:
We should update the development status classifier to 4 - beta....`Development Status :: 4 - Beta` to reflect the updated package version and pre-release segment.